javascript/builtins/Intl.json

```diff
@@ -708,14 +708,21 @@
                   "version_added": false
                 },
                 "chrome": {
-                  "version_added": "55",
-                  "flag": {
-                    "type": "compile_flag",
-                    "name": "--datetime-format-to-parts"
-                  }
+                  {
+                    "version_added": "57"
+                  },
+                  {
+                    "version_added": "55",
+                    "version_removed": "60",
+                    "flag": {
+                      "type": "preference",
+                      "name": "--datetime-format-to-parts"
+                    }
+                  },
+                  "notes": "Enabled by default in 57. The flag was removed in 60"
```

Comment:
This note is redundant, please remove it.

Reply:
I can, but without notes, I'm afraid it's not clear in what version it's controlled by a flag and in what version it's enabled unconditionally.

Reply:
We're generating some notes from the data and we're experimenting with a new compat table layout to address this. Did you see our beta tables? https://discourse.mozilla.org/t/beta-testing-new-compatability-tables/21269